src/Confluent.Kafka/Admin/DeleteGroupsException.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright 2022 Confluent Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Refer to LICENSE for more information.
+
+using System;
+using System.Linq;
+using System.Collections.Generic;
+
+
+namespace Confluent.Kafka.Admin
+{
+    /// <summary>
+    ///     Represents an error that occured during a delete groups request.
+    /// </summary>
+    public class DeleteGroupsException : KafkaException

```

Comment:
In the Java client, the method is deleteConsumerGroups.
We should follow the same naming convention in .NET - here `DeleteConsumerGroupsException`.

Reply:
hmm, or should we? I expect the deletegroups protocol request works on any group type ... maybe keep it as DeleteGroups.

Reply:
Looking at the Kafka protocol it is DeleteGroups
https://kafka.apache.org/protocol.html#The_Messages_DeleteGroups

I would say it would be best naming to align with the Kafka Protocol - also delegates which invokes LibrdKafka methods doesn't have Consumer in naming.

Reply:
I agree, but the analogous thing to the java client would be to expose the DeleteGroups protocol method as DeleteConsumerGroups. Did they have a good reason for it? That is what i'm wondering (leave as is for now, i'll research it)